mlfinlab/ensemble/sb_bagging.py

```diff
@@ -634,29 +597,3 @@ def _validate_estimator(self):
         """Check the estimator and set the base_estimator_ attribute."""
         super(BaggingRegressor, self)._validate_estimator(
             default=DecisionTreeRegressor())
-
-    def _set_oob_score(self, X, y):
-        n_samples = y.shape[0]
-
-        predictions = np.zeros((n_samples,))
-        n_predictions = np.zeros((n_samples,))
-
-        for estimator, samples, features in zip(self.estimators_,
-                                                self.get_estimators_samples_(),
```

Comment:
Are you sure you can delete this code entirely? I see that you are calling the custom method `self.get_estimators_samples_()` which is not present in the parent class.

Reply:
I've changed method naming to sklearn's so now it refers correctly